packages/@react-spectrum/combobox/src/ComboBox.tsx

```diff
@@ -118,14 +122,39 @@ const ComboBoxBase = React.forwardRef(function ComboBoxBase<T extends object>(pr
 
   let style = {
     ...overlayProps.style,
-    width: menuWidth
+    width: isQuiet ? null : menuWidth,
+    minWidth: isQuiet ? `calc(${menuWidth}px + calc(2 * var(--spectrum-dropdown-quiet-offset)))` : menuWidth
   };
 
+  let labelClassName;
+  let inputClassName;
+  if (label && isQuiet) {
+    labelClassName = classNames(
+      labelStyles,
+      'spectrum-FieldLabel--quiet'
+    );
+
+    inputClassName = classNames(
+      labelStyles,
+      'spectrum-Field-fieldInput--quiet'
+    );
+  }
```

Comment:
For the 28px -> 20px spacing between label and bottom of combobox input text design feedback for quiet comboboxes